src/components/Dao/DaoRedemptionsPage.tsx

```diff
@@ -29,6 +30,59 @@ class DaoRedemptionsPage extends React.Component<IProps, null> {
     const proposalsHTML = proposals.map((proposalData: any) => {
       const proposal = new Proposal(proposalData.id, arc);
 
+      // hack to work around https://github.com/daostack/subgraph/issues/304
+      let beneficiaryHasRewards = false;
+      const proposalState = proposalData;
```

Comment:
why do we need this?

Reply:
we'll remove all this horrible code as soon as the subgraph bug is fixed